bin/hardening/1.1.1.1_disable_freevxfs.sh

```diff
@@ -17,28 +17,43 @@ HARDENING_LEVEL=2
 # shellcheck disable=2034
 DESCRIPTION="Disable mounting of freevxfs filesystems."
 
-# Note: we check /proc/config.gz to be compliant with both monolithic and modular kernels
-
-KERNEL_OPTION="CONFIG_VXFS_FS"
-MODULE_NAME="freevxfs"
+FSTYPE_NAME="freevxfs"
```

Comment:
You forgot to change 1.1.1.23, 3.4.1, 3.4.2, 3.4.3, 3.4.4 that use the same function `is_kernel_option_enabled`

Reply:
You have right, I propose to finish discussion on remaining point to decide which kind of final fix should be done

Reply:
fixed in 678425dab5d62da84f095c4bbe62abf8646ea163 I update "tests set" to be consistent (not checked on docker)